src/window.py

```diff
@@ -206,6 +206,8 @@ def closeEvent(self, event):
         self.close_signal.emit()
         event.accept()
 
+        self.centralWidget().layout().itemAtPosition(0, 0).widget().Finalize()
```

Comment:
can we find vtk widget somehow simpler? 
If it is widget3d - we can save it to the self object and then use it here.